gordo_components/builder/build_model.py

```diff
@@ -73,34 +80,53 @@ def build_model(
     logger.debug(f"Initializing Model with config: {model_config}")
     model = serializer.pipeline_from_definition(model_config)
 
-    # Cross validate
-    logger.debug(f"Starting to do cross validation")
-    start = time.time()
+    cv_duration_sec = None
 
-    scores: Dict[str, Any]
-    if hasattr(model, "predict"):
-        cv_scores = cross_val_score(
-            model,
-            X,
-            y,
-            scoring=make_scorer(metric_wrapper(explained_variance_score)),
-            cv=TimeSeriesSplit(n_splits=3),
-        )
-        scores = {
-            "explained-variance": {
-                "mean": cv_scores.mean(),
-                "std": cv_scores.std(),
-                "max": cv_scores.max(),
-                "min": cv_scores.min(),
-                "raw-scores": cv_scores.tolist(),
+    if cv_mode.lower() in ("cross_val_only", "full_build"):
+
+        # Cross validate
+        logger.debug(f"Starting to do cross validation")
```

Comment:
`Starting cross validation` sounds more correct to me, but I be not native freedom-speaker. @milesgranger?